packages/console/src/scss/normalized.scss

```diff
@@ -1,8 +1,10 @@
 body {
   margin: 0;
   padding: 0;
-  font-family: sans-serif;
+  font-family: 'SF UI Text', 'SF Pro Display', sans-serif;
```

Comment:
Should at least include `Segoe UI`, `Helvetica` and `Roboto`, and maybe even some emoji supports like `Apple Color Emoji` and `Segoe UI Emoji`

Reply:
it's not fixed for now, we need a session for this. you can add an issue for guamian and eric

Reply:
ok